DQM/SiStripMonitorClient/plugins/SiStripDaqInfo.cc

```diff
@@ -22,6 +25,52 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <map>
+
+class SiStripFedCabling;
+class TrackerTopology;
+
```

Comment:
Is this the right fix?
I think this should be moved to `DQMEDHarvester` instead.